editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -447,10 +457,15 @@ function renderJSXElement(
 
   if (elementPath != null && validPaths.has(EP.makeLastPartOfPathStatic(elementPath))) {
     if (elementIsTextEdited) {
-      const textBlock = childrenWithNewTextBlock.find(
-        (c): c is JSXTextBlock => c.type === 'JSX_TEXT_BLOCK',
-      )
-      const textContent = unescapeHTML(textBlock?.text ?? '')
+      const text = childrenWithNewTextBlock
+        .filter(
+          (c): c is JSXTextBlock =>
+            c.type === 'JSX_TEXT_BLOCK' ||
+            (c.type === 'JSX_ELEMENT' && c.name.baseVariable === 'br'),
+        )
```

Comment:
this might go into a separate function, it's general enough that it might be useful somewhere else as well

Reply:
https://github.com/concrete-utopia/utopia/pull/3048/commits/a83181f55998e46598c500f06584e94970bc59c9